tests/Client/RequestTest.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types=1);
+
+namespace ekinhbayar\GitAmpTests\Client;
+
+use Amp\Artax\Client;
+use ekinhbayar\GitAmp\Client\GitAmp;
+use ekinhbayar\GitAmp\Events\Factory;
+use PHPUnit\Framework\TestCase;
+
+class RequestTest extends TestCase
```

Comment:
`class GitAmpTest extends TestCase` this way it's easier to find out what class it belongs to when something fails.